certified-connectors/EXPOCAD/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#e30613",
+    "capabilities": [],
+    "publisher": "Michael Ochs"
```

Comment:
@mikefactorial The `publisher` property value should not be an individual name. Can you please provide the responsible company or org name.

Documentation - [Adding metadata](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#adding-metadata)

Reply:
Hi @sriyen-msft this has been resolved in the latest commit